File: WEEK1/DAY6/Prog2/P2.c
#include <stdio.h>
int main(){
    int N;
    printf("Find product of digits of N\n");
    printf("Enter N: ");
    N=scanf("%d",&N);
    int ans=1;
    int n=N;
    int dig;
    while(n>0){
        dig=n%10;
        ans*=dig;
        n/=10;
    }
    printf("Product of digits of %d = %d\n",N,ans);
}

File: WEEK1/DAY6/Prog2/P2.java
import java.util.*;
public class P2 {
    public static void main(String[] args) {
        int N;
        Scanner sc=new Scanner(System.in);
        System.out.println("Find product of digits of N");
        System.out.print("Enter N: ");
        N=sc.nextInt();
        int n=N;
        int dig;
        int ans=1;
        while(n>0){
            dig=n%10;
            ans*=dig;
            n/=10;
        }
        System.out.println("Product of digits of "+N+" = "+ans);
        sc.close();
    }
}
